pkg/printers/template.go

```diff
@@ -112,3 +116,11 @@ func (p *TemplatePrinter) safeExecute(w io.Writer, obj interface{}) error {
 	}
 	return retErr
 }
+
+func base64decode(v string) string {
+	data, err := base64.StdEncoding.DecodeString(v)
```

Comment:
Does go json encoding use StdEncoding for []byte serialization?

Reply:
Yes. 
![screen shot 2018-03-06 at 7 28 46 pm](https://user-images.githubusercontent.com/372794/37066717-afcc9d70-2174-11e8-9b02-5b729fe8f72c.png)